Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,58 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 support new features in MongoDB 4.4. These features include:
+
+* Support for all new
+  [``$meta``](https://docs.mongodb.com/manual/reference/operator/projection/meta/)
+  projections: `randVal`, `searchScore`, `searchHighlights`,
+  `geoNearDistance`, `geoNearPoint`, `recordId`, `indexKey` and
+  `sortKey`
+* Support for passing a hint to update commands as well as
+  `findAndModify` update and replace operations
+* Support for `allowDiskUse` on find operations
+* Support for `MONGODB-AWS` authentication using Amazon Web Services
+  (AWS) Identity and Access Management (IAM) credentials
+* Support for stapled OCSP (Online Certificate Status Protocol) (macOS only)
+
+Other new additions and updates in this beta include:
+
+* A new target of .NET Standard 2.0
+* Support for Snappy compression on .NET Core on Windows
```

Comment:
This sounds like we only support .NET Core. Should we say

    .NET Core (in addition to existing support on .NET Framework)

Reply:
Done

Reply:
Why did this comment disappear from the Files tab?